library/pod-security-policy/privileged-containers/template.yaml

```diff
@@ -16,14 +16,27 @@ spec:
 
         violation[{"msg": msg, "details": {}}] {
             c := input_containers[_]
-            c.securityContext.privileged
+            input_allow_privileged_container(c)
             msg := sprintf("Privileged container is not allowed: %v, securityContext: %v", [c.name, c.securityContext])
```

Comment:
if `c.securityContext` is undefined, this line will not match, which would render the changes you made moot.

Reply:
thank you so much for warning me about this @maxsmythe , I added unit tests and fixed the message to avoid this problem.